### PR TITLE
detail-page-css-Esteban-Code

### PR DESCRIPTION
**Pull request by melaniearista**

### CODE REVIEW

File: src/styles/productDetail.css [hunk 1]
```diff
@@ -1,90 +1,72 @@
...
-    border-radius: 80px;
-    border: #333;
-    border-radius: 8px;
-    background-color: #CBD1E4;
-    width: 150px;
-    text-align: center;
-    margin-left: 10px;
-    font-family: 'Calibri Light', sans-serif;
-  }
+#titulo-categorias {
```
Comment (PedroDCA) [inline src/styles/productDetail.css:14]: Could we keep the style names in english?

File: src/components/productDetail.js [hunk 2]
```diff
@@ -11,28 +12,28 @@ function ProductDetail({ productInformation }) {
...
+            <p id="categories">Categoria 1</p>
+            <p id="categories">Categoria 2</p>
           </div>
         </div>
-        <h2>{productInformation.name}</h2>
-        <div>
-          <p>Star section</p>
+        <h2 className="titulo-producto">{productInformation.name}</h2>
+        <div className="star-section">
+          <p>★★★★☆</p>
```
Comment (PedroDCA) [inline src/components/productDetail.js:29]: Is this value changing when the ratingAverage is different than 4?

File: src/components/productDetail.js [hunk 3]
```diff
@@ -11,28 +12,28 @@ function ProductDetail({ productInformation }) {
     ? "Agregar al carrito"
     : "En carrito";
   return (
-    <div className="d-flex">
+    <div className="product detail d-flex">
       <div>
-        <img src={productInformation.imageUrl} alt="Product" />
+        <img src={productInformation.imageUrl} alt="Product" className="imgb"/>
```
Comment (PedroDCA) [inline src/components/productDetail.js:17]: What means `imgb`? Could we keep a better naming for this class?

**Review by PedroDCA — CHANGES_REQUESTED** [review]

File: src/components/productDetail.js (hunk 2 above)
Comment (LaNerva) [reply src/components/productDetail.js:29]: That rating was coded for the vendor section, not for the product rating. The stars are simulating the rating

Comment (PedroDCA) [reply src/components/productDetail.js:29]: Could you implement the css to be able to update the stars according to the rating? On this component you have available the `productInformation.ratingAverage` variable to know how many stars you have to show.

**Review by PedroDCA — CHANGES_REQUESTED** [review]

**Review by PedroDCA — APPROVED** [review]
Approving pull request to have progress, but it is pending the changes on the next pull request

**Pull request merged**